src/sentry/static/sentry/app/index.js

```diff
@@ -150,7 +150,6 @@ export default {
     PluginConfig: require('./components/pluginConfig').default,
     ProjectIssueTracking: require('./views/projectIssueTracking').default,
     ProjectSelector: require('./components/projectHeader/projectSelector').default,
-    RuleEditor: require('./views/ruleEditor').default,
```

Comment:
I doubt it's in getsentry, but I would double check just in case.

Reply:
Yep, it's not